schedule.js

```diff
@@ -0,0 +1,21 @@
+import schedule from 'node-schedule';
+import childProcess from 'child_process';
+
+const rule = new schedule.RecurrenceRule();
+rule.minute = 30; // every hour at 30 minutes after the hour
```

Comment:
Isn't there a way to say “every hour starting from now”?

Reply:
Simply with `setInterval`, but I'm not sure that what we want. I think is better to have predictable runs.